video/out/wayland_common.c

```diff
@@ -1119,6 +1121,8 @@ int vo_wayland_init(struct vo *vo)
         return false;
     }
 
+    wl->opts = mp_get_config_group(wl, wl->vo->global, &wayland_conf);
```

Comment:
I had to move this up because I needed the options to be processed in order to access the app id in `create_xdg_surface`. I'm not sure if there are any negative consequences by doing this but in my trivial tests I didn't see anything out of the ordinary happen as a result. I'm also not sure if I should've moved up the whole block or just the single line but I felt like it was less risky to just move the line I needed.